CHANGELOG.md

```diff
@@ -6,23 +6,28 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.3.2] - unreleased
 
+### Changed
+
+- public strategies are all functions to be run to get the runtime function
+  `iso8601Date` => `iso8601Date()`
```

Comment:
Is this just for consistency, since before some of them were functions and some of them returned functions?

Reply:
Exactly yes.